inspectors/doj.py

```diff
@@ -18,11 +19,13 @@
 from datetime import datetime
 from utils import utils, inspector
 import logging
+from urllib.parse import urljoin, urlparse
+import os
 
 # accumulates information on reports as they're seen
 report = {}
 
-base_url = "http://www.justice.gov"
+base_url = "https://oig.justice.gov/reports/"
```

Comment:
Another [notch in the belt](http://www.washingtonpost.com/blogs/the-switch/wp/2015/04/16/why-confidential-tips-to-the-government-may-not-be-confidential-after-all/ for @csoghoian. :smiley: